GVRf/Framework/backend_oculus/src/main/jni/ovr_view_manager.cpp

```diff
@@ -29,7 +29,13 @@ void Java_org_gearvrf_OvrViewManager_cull(JNIEnv * jni, jclass clazz,
     Scene* scene = reinterpret_cast<Scene*>(jscene);
     Camera* camera = reinterpret_cast<Camera*>(jcamera);
     ShaderManager* shader_manager = reinterpret_cast<ShaderManager*>(jshader_manager);
-    Renderer::cull(scene, camera, shader_manager);
+    renderer = Renderer::getInstance();
```

Comment:
Can Renderer::getInstance() really return nullptr? I think new should throw; in that case the checks and the logs are unnecessary.